user/Makefile

```diff
@@ -10,8 +10,8 @@ APP := nfp-user.out
 
 all: $(APP)
 
-CFLAGS := -g3 -Wall -Werror -Wno-format-truncation -I$(DIR) -I$(DIR)/lib/ -I$(DIR)/nfpcore/ -MMD -MP
-LDLIBS := -lm
+CFLAGS := -g3 -Wall -Wno-format-truncation -I$(DIR) -I$(DIR)/lib/ -I$(DIR)/nfpcore/ -MMD -MP # -Werror
+LDLIBS := -lm -lpthread
```

Comment:
Add "-pthread" flag to CFLAGS
Refer Compiling on Linux section here: http://man7.org/linux/man-pages/man7/pthreads.7.html